diff --git a/src/lib/base-make.sh b/src/lib/base-make.sh
index 8a6e070..bfae3f6 100755
--- a/src/lib/base-make.sh
+++ b/src/lib/base-make.sh
@@ -58,12 +58,23 @@ function lowerstrap-systemd-nspawn() {
   echo "PACKAGER=\"${CAUR_PACKAGER}\"" | tee -a './etc/makepkg.conf'
   install -m755 "$CAUR_GUEST"/bin/* './usr/local/bin/'
 
-  stee -a './etc/pacman.conf' <<EOF
-
-[${CAUR_DB_NAME}]
-SigLevel = Never
-Server = ${CAUR_URL}
-
+  gawk -i inplace -f - './etc/pacman.conf' <<EOF
+  {
+    if (\$0 == "[options]")
+    {
+        print
+        next
+    }
+  }
+  /^\[[^ \[\]]+\]/ {
+    if (!set) {
+        print "[${CAUR_DB_NAME}]"
+        print "Server = ${CAUR_URL}"
+        print ""
+        set=1
+    }
+  }
+  1
 EOF
 
   echo 'en_US.UTF-8 UTF-8' | stee './etc/locale.gen'
